docs/UsingCppInterOp.rst

```diff
@@ -65,35 +65,245 @@ robust (simple function calls, no inheritance, etc.). The goal is to make it as
 close to the compiler API as possible, and each routine should do just one thing.
 that it was designed for.
 
-Further Enhancing the Dynamic/Automatic bindings in CPPYY
-=========================================================
-The main use case for CppInterOp is the CPPYY service. CPPYY is an
-automatic run-time bindings generator for Python and C++, and supports a wide
-range of C++ features (e.g., template instantiation). It operates on demand and
+How cppyy relates to CppInterOp and how it works
+================================================
+
+CppInterOp ties into several `enhancements in cppyy`_, following are a few:
+
+1. **CppInterOp is a cppyy-inspired Library** that enables interoperability
+   with C++ code, bringing the speed and efficiency of C++ to simpler, more
+   interactive languages like Python.
+
+2. **Reducing dependencies** of cppyy to minimize code bloat and make it
+   faster.
+
+3. **LLVM Integration**: CppInterOp's integration with LLVM's Clang-REPL
+   component (that can then be used as a runtime compiler for CPPYY), will
+   further enhance CPPYY's performance.
+
+**CppInterOp is a cppyy-inspired Library**
```

Comment:
perhaps this can all come before where I commented above? Then CppInterOp is introduced and explained before we talk about cppyy?

Reply:
outdated: rephrased the whole section from an "**enhancement in cppyy**" stance to "**How cppyy leverages CppInterOp**".